docs/src/contributing/git-workflow.md

```diff
@@ -51,36 +99,173 @@ If the branch exists in a triton/neptune official repo, (as opposed to a persona
+
+resets minor and patch to 0.
+
+**minor**: bumped any time changes from master are included in a release candidate.
+
+resets patch to 0.
+
+**patch**: bumped any time changes from a release branch are included in a release.
+
+resets candidate to "-rc1".
+
+**candidate**: added/bumped when a new public testnet is created for a candidate release.
+
+*Note that the optional suffix is supported by cargo and related tools.*
+
+## typical release
+
+The normal/typical release flow would be:
+
+1. normal changes (not consensus) are introduced in master.
+2. a release candidate is tagged from master, eg `v0.3.0-rc1` and built.
+3. a testnet is created for `v0.3.0-rc1`
+4. if significant problems are found then a new branch `v0.3.0-rc2` is created at `v0.3.0-rc1`, or possibly from master.  commit(s) are added. When ready, the final commit in the branch is tagged with `v0.3.0-rc2`, another testnet created, and so on.
+5. fixes for the release-candidate should also be applied to master, if applicable.
+6. After two weeks, or when team decides, the latest release-candidate tag is also tagged with `v0.3.0`, signifying an actual release.
+7. The release is performed.
+
+*note that by the time the release actually occurs, master may have diverged significantly.*
+
+The release flow for a major version change is essentially the same except that the major version number is bumped and the minor version returns to 0.
+
+
+## Release Candidate Protocol  (-rc1, -rc2, etc)
+
+ - determine source branch and new version.  (see visualization above.)
+ - checkout the source branch and ensure working directory is clean.
+ - Set the version in `Cargo.toml` (including the -rcX suffix).
+ - Ensure that source branch (eg master) builds against crates that live on
+   [crates.io](https://crates.io). In particular, no dependencies on github
+   repositories or revisions.
+ - review `README.md` to ensure the installation instructions are up-to-date.
  - Ensure that all tests pass.
- - Bump the version in `Cargo.toml`
- - Create a commit with the subject line `v0.0.6` (or watever the new version number is) and in the body list all the changes.
- - Push to `master` on github.
- - Add a tag marking the current commit with the version:
-   - `git tag v0.0.6` (or whatever the next version is)
+ - Create a commit with the subject line `v0.3.0-rc1` (or whatever the new version number is) and in the body list all the changes.
+ - Push present branch to github.
+ - Add a tag marking the current commit version:
+   - `git tag v0.3.0-rc1` (or whatever the next version is)
+   - `git push --tags`.
+ - build binaries and make a "release" on github
+ - start a testnet with the release-candidate executables.
+ - announce the release and new testnet on [talk.neptune.cash](https://talk.neptune.cash).
+
+## Release Protocol (major.minor.point)
+  Perform steps as above except:
+   - `git tag v0.3.0` (or whatever the next version is)
+   - `git tag latest-release`
    - `git push --tags`.
```

Comment:
Also here. I think this should work instead: `git push origin tag v0.3.0 tag latest-release`

Reply:
addressed in 66b171787d18c6ad4aaf45854923545d15666d28